File: Server/Tuckshop.IdentityServer/Tuckshop.IdentityServer.Contracts/Models/UserManagement/Queries/UserManagementActionLogLookupCriteria.cs
namespace Tuckshop.IdentityServer.Contracts.UserManagement.Queries
{
  /// <summary>
  /// Criteria for the user management action log lookup.
  /// </summary>
  public class UserManagementActionLogLookupCriteria
  {
    /// <summary>
    /// Gets or sets the user id.
    /// </summary>
    public string UserId { get; set; } = string.Empty;
  }
}

File: Client/Tuckshop.WebApp/src/Identity/Models/UserManagement/Queries/UserManagementActionLogLookupCriteria.ts
import { NeoModel, Validation, ValueObject } from '@singularsystems/neo-core';

@NeoModel
export default class UserManagementActionLogLookupCriteria extends ValueObject {

    public userId: string = "";

    // Client only properties / methods

    protected addBusinessRules(rules: Validation.Rules<this>) {
        super.addBusinessRules(rules);
    }

    public toString(): string {
        return "User Management Action Log Lookup Criteria";
    }
}